tooling/flake8.config

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E113,E121,E122,E123,E126,E127,E128,E129,E202,E203,E221,E225,E226,E228,E231,E241,E251,E261,E265,E301,E302,E303,E402,E501,E901,F841,W292,W391,W503,W602
```

Comment:
Good config. It made some good catches and didn't make much counterproductive noise at all.